conditional/blueprints/major_project_submission.py

```diff
@@ -69,6 +69,10 @@ def submit_major_project(user_dict=None):
         return jsonify({"success": False}), 400
     project = MajorProject(user_dict['username'], name, description)
 
+    # Don't you dare try pinging @channel
+    if "@" in name:
+        name = "@ ".join(name.split("@"))
```

Comment:
This is not how slack bots @ channel or @ announcements

Please research slack bot api